rails/strings.rb

```diff
@@ -692,7 +692,8 @@ def strings
                 machine_type_notes: {
                     gen1: "The tried and tested machines that have been serving our customers for many years now",
                     gen2: "These machines run on a new hardware generation, offering improved performance compared to Gen1 hardware"
-                }
+                },
+                m1_elite_xl_beta_warning: "M1 Elite XL machines are in beta. Some features, like the Android emulator, don't work on it yet."
```

Comment:
Is this the official, signed-off copy? I'd say "supported", not "work", but it's not a blocker :) 

```suggestion
                m1_elite_xl_beta_warning: "M1 Elite XL machines are in beta. Some features (e.g. the Android emulator) are not supported yet."
```